copy-images.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
This is a nice utility, but I don't think it belongs in the root, does it?

Reply:
true, i will move it to a better place